CONTRIBUTING.md

```diff
@@ -51,10 +51,6 @@ further discussion._
 
 _First time contributors should add their name to [AUTHORS](../AUTHORS)._
 
-_Stockfish's development is not focused on adding new features. Thus any pull
-request introducing new features will potentially be closed without further
```

Comment:
Why remove this?

Reply:
My mistake, I thought it was a duplicate sentence but the other sentence replaces pull request with issue.